daffodil-cli/src/it/scala/org/apache/daffodil/saving/TestCLISaveParser.scala

```diff
@@ -262,6 +262,10 @@ class TestCLISaveParser {
     }
   }
 
+  /**
+   * Note that in Daffodil 2.6.0 behavior of external variables changed. They are not saved as part of binary
+   * compiling. They are a runtime-thing only.
+   */
```

Comment:
Maybe we shoudl add a warning in the CLI

Reply:
There are things we can check at compile time. First is that the settings are going to be legal, which requires that the variables being set are in fact declared 'external', and that the value provided can be converted to the proper type. 

A warning, that the external vars are checked, but not saved, as part of save-parser makes sense at the CLI and at the save API call. Internally, however, where we save/restore (e.g., for every TDML test) we don't want this warning.